### DISCUSSION TITLE
Quarto project as part of existing RStudio project

### DISCUSSION THREAD

**Discussion by wiesehahn** [quarto-dev/quarto-cli#1988] [Q&A] [answered]
Hi, 
I would like to create a Quarto book or website for documenting a project which is already existing as RStudio project. 
How would I do this / Whats the best-practice?

I can simply initialize e.g. a Quarto book project within the RStudio project but this seems not right. I didnt think too much about the advantages/disadvantages but I think it makes more sense to manage the project as one.

**Comment by jjallaire** [comment, marked as answer]
Yes, you can just add a `_quarto.yml` file to the directory defining the book and next time you start RStudio it will be recognized as a Quarto project.
